src/tests/suite/daos_obj.c

```diff
@@ -3295,7 +3295,10 @@ io_pool_map_refresh_trigger(void **state)
 	struct ioreq	req;
 	d_rank_t	leader;
 
-	/** choose random object */
+	/* needs at lest 2 targets */
+	if (!test_runable(arg, 2))
+		skip();
+
 	test_get_leader(arg, &leader);
```

Comment:
seems better to check its return value, when it failed it will not set the leader.

BTW, don't use rank 0 as pool service leader seems is just a current limitation just because previously (long time ago) cart level cannot handle fault of rank 0, maybe can release the check to be leader >= 0.